libraries/plugins/apis/network_broadcast_api/network_broadcast_api_plugin.cpp

```diff
@@ -11,6 +11,8 @@ void network_broadcast_api_plugin::set_program_options( options_description& cli
 void network_broadcast_api_plugin::plugin_initialize( const variables_map& options )
 {
    api = std::make_shared< network_broadcast_api >();
+   FC_ASSERT( !appbase::app().get_plugin< rc::rc_plugin >().get_rc_plugin_skip_flags().skip_reject_not_enough_rc,
```

Comment:
I am assuming this addition in particular is just until the transaction confirmation API is implemented?